### DISCUSSION TITLE
Issue on following Blog Tutorial

### DISCUSSION THREAD

**Discussion by ryanadhibit** [remix-run/remix#2925] [Q&A] [answered]
I have issue following blog tutorial https://remix.run/docs/en/v1/tutorials/blog

I already run `npm add @types/marked -D`  and it is there in my `package.json` `"@types/marked": "^4.0.3"`

But somehow I cannot run my code and receive error like this 

![Screenshot 2022-04-20 092944](https://user-images.githubusercontent.com/70564966/164135109-9d81ed7a-4873-4939-baf9-f765c995e141.png)

Any suggestion?

**Comment by machour** [comment]
Did you also install the actual package?

```
npm add marked    <---------- you seem to have forgotten this one
# if using typescript
npm add @types/marked -D
```

**Comment by ryanadhibit** [comment]
@machour I installed it here since I am using typescript
![Screenshot 2022-04-20 094034](https://user-images.githubusercontent.com/70564966/164584197-b0c8ffda-9e50-4f17-b613-eef3d7a556ba.png)

**Comment by machour** [comment, marked as answer]
@types/marked is an *additional* package to install to get typings for this library. You still need to install `marked`. 

I'll clarify the comment in the documentation!

**Reply by ryanadhibit** [reply]
great it works now. thanks
